README.md

```diff
@@ -56,6 +56,40 @@ All assumptions, data and sources used are available in the [methodology](https:
 
 ### Prerequisites
 
+Have docker installed
+
+### Launch the app
+
+At the root of the directory, run:
+
+```bash
+docker-compose up --build
+```
+
+You can now connect to [http://localhost:3000](http://localhost:3000) to test the application.
+
+Note: If you want to launch them in the backgroup and leave them running, you can add the flag `-d` or `--detach`.
+
+### Rebuild and restart a service
+
+At the root of the directory, run:
```

Comment:
```suggestion
In `/frontend`, copy `.env.dist` and rename it `.env`. Then at the root of the directory, run:
```

Reply:
Actually no, I have changed my mind. API_URL should have a default value in `frontend/src/config.ts`, can you modify it like this:
```
export const API_URL = process.env.REACT_APP_API_URL ?? "http://localhost:8000";
```

No need to update the readMe

Reply:
Done in last commit!